code/modules/organs/organ_external.dm

```diff
@@ -161,8 +156,16 @@
 	brute *= brute_mod
 	burn *= burn_mod
 
+	// Threshold needed to have a chance of hurting internal bits with something sharp
+	var/const/sharp_thresh_int_dmg = 5
+	// Threshold needed to have a chance of hurting internal bits
+	var/const/thresh_int_dmg = 10
+	// Probability of taking internal damage from sufficient force, while otherwise healthy
+	var/const/dmg_prob = 5
+	// Probability of taking internal damage when hit while lacking encasing bones
+	var/const/no_bone_dmg_prob = 30
 	// High brute damage or sharp objects may damage internal organs
-	if(internal_organs && (brute_dam >= max_damage || (((sharp && brute >= 5) || brute >= 10) && prob(5))))
+	if(internal_organs && (brute_dam >= max_damage || (((sharp && brute >= sharp_thresh_int_dmg) || brute >= thresh_int_dmg) && prob(dmg_prob)) || (!encased && prob(no_bone_dmg_prob))))
```

Comment:
This is so organs without an `encased` provide less protection to the organs inside.